.gitignore

```diff
@@ -77,5 +77,6 @@ yarn.lock
 
 #IDE Files
 .vscode
+access.txt
```

Comment:
@Sayantan-s, this is probably unnecessary. Will you take a moment to remove this?

Reply:
@leecalcote yeah, I'll remove this rn